.bouncer.yaml

```diff
@@ -11,6 +11,9 @@ ignore-packages:
   # packageurl-go is released under the MIT license located in the root of the repo at /mit.LICENSE
   - github.com/anchore/packageurl-go
 
+  # crypto/internal/boring is released under the openSSL license as a part of the Golang Standard Libary
```

Comment:
I thought we already covered this license, but I had to re-add since static analysis is failing